mongodbatlas/data_source_mongodbatlas_advanced_clusters.go

```diff
@@ -24,6 +25,7 @@ func dataSourceMongoDBAtlasAdvancedClusters() *schema.Resource {
 				Computed: true,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
+						"advanced_configuration": clusterAdvancedConfigurationSchemaComputed(),
 						"backup_enabled": {
```

Comment:
This is not related to this change but I see that this schema has
```
						"name": {
							Type:     schema.TypeString,
							Computed: true,
						},
```
where we are using `Required: true` for the `name` in [data_source_mongodbatlas_advanced_cluster.go](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/50b647f190a38a0fd5081d7fa36d95ded9292577/mongodbatlas/data_source_mongodbatlas_advanced_cluster.go#L83). 
Is this correct or both should be `Computed: true`?

Reply:
In the singular datasource is required to get a specific advanced cluster along the project ID, in the plural datasource, it will get all the advanced clusters from the project ID provided and then list it with all computed parameters.